uxarray/helpers.py

```diff
@@ -250,16 +241,15 @@ def get_all_face_area_from_coords(x,
     """
     num_faces = face_nodes.shape[0]
     area = np.zeros(num_faces)  # set area of each face to 0
-
     for i in range(num_faces):
-
+        # print(i, face_nodes[i])
         face_z = np.zeros(len(face_nodes[i]))
 
-        face_x = x[face_nodes[i]]
-        face_y = y[face_nodes[i]]
+        face_x = x[face_nodes[i].astype(np.int64)]
+        face_y = y[face_nodes[i].astype(np.int64)]
         # check if z dimension
         if dim > 2:
-            face_z = z[face_nodes[i]]
+            face_z = z[face_nodes[i].astype(np.int64)]
```

Comment:
Might be better to set the type to `int` outside of the loop, instead of doing it while indexing